region/maridia/outer/Mama Turtle Room.json

```diff
@@ -105,13 +113,23 @@
       "to": [
         {"id": 2},
         {"id": 3},
-        {"id": 5}
+        {"id": 5},
+        {"id": 6}
       ]
     },
     {
       "from": 5,
       "to": [
-        {"id": 2}
+        {"id": 2},
+        {"id": 6}
+      ]
+    },
+    {
+      "from": 6,
+      "to": [
+        {"id": 3},
+        {"id": 5},
```

Comment:
Adding [6, 2] would seem more useful than [6, 5] to me.
And actually we would want [6, 2] strats since you can reach 3 itemless, then use that item to continue to 2 from 6 in Not-MapRando logic.

Reply:
At one level that makes sense to me.  But the problem I'm having is really with node 2, I think.  Unfortunately, standing on top of the item block vs. hanging below the grapple block is not really the same thing... but there's only a single node here.  Maybe I'm crazy and I'm the only person who has trouble transitioning from below->above the grapple block?  If you guys are saying "grappling from below->above the item block is a base strat that anyone can do" I'm fine being overruled.  But I will tell you that I could grapple back and forth between 5 and 6 like 20 times in a row without falling, but trying to swing up onto the block and land is a big challenge for me.